### ISSUE TITLE
checkbox: high contrast focus indicator not visible

### ISSUE THREAD

**Issue by devversion** [angular/components#19443]
The alternate high contrast focus in the checkbox no longer seems to work due recent changes to the `cdk-high-contrast` mixin. It looks like the generated CSS selector is incorrect:

```scss
.mat-checkbox.cdk-keyboard-focused .cdk-high-contrast-active .mat-checkbox-frame {
  border-style: dotted
}
```

It might be worth checking if other components are affected too.
